src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import './FindMovie.scss';
-
+import { getMovie } from '../../api/api';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(null);
+  const [error, setError] = useState('');
+
+  const updateTitle = (newTitle) => {
+    setTitle(newTitle);
+    if (newTitle) {
+      setError('');
+    }
+  };
+
+  const findMovie = async() => {
+    if (title) {
+      const foundMovie = await getMovie(title);
+
+      if (foundMovie.Error) {
+        setError(foundMovie.Error);
+        setMovie(null);
+      } else {
+        const newMovie = {
+          title: foundMovie.Title,
+          description: foundMovie.Plot,
+          imgUrl: foundMovie.Poster,
+          imdbUrl: `https://www.imdb.com/title/${foundMovie.imdbID}`,
+          imdbId: foundMovie.imdbID,
+        };
+
+        setMovie(newMovie);
+        setError('');
+      }
+    } else {
+      setMovie(null);
+      setError('Not title entered');
+    }
```

Comment:
краще робити, щось типу цього, щоб не було вкладених if
```suggestion
    if (!title) {
       setMovie(null);
      setError('Not title entered');
      
      retrun null;
    }
    
    const foundMovie = await getMovie(title);

    if (foundMovie.Error) {
      setError(foundMovie.Error);
      setMovie(null);
    } else {
      const newMovie = {
        title: foundMovie.Title,
        description: foundMovie.Plot,
        imgUrl: foundMovie.Poster,
        imdbUrl: `https://www.imdb.com/title/${foundMovie.imdbID}`,
        imdbId: foundMovie.imdbID,
      };

      setMovie(newMovie);
      setError('');
    }
```